src/posix/platform/resolver.cpp

```diff
@@ -71,6 +74,15 @@ void Resolver::Init(void)
     LoadDnsServerListFromConf();
 }
 
+void Resolver::Setup(void)
+{
+    OT_ASSERT(gInstance != nullptr);
+
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
+    otBorderRoutingSetRdnssAddrCallback(gInstance, &Resolver::BorderRoutingRdnssCallback, this);
+#endif // OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
```

Comment:
We need to add comment in `#endif` only when there are more then 20 lines from the corresponding `#if`. Suggest removing this. This is from our [OpenThread style guide](https://github.com/openthread/openthread/blob/d095eb38696adedccc5d19497baa2e9061a93bee/STYLE_GUIDE.md#language-independent)

>  - All `#endif` directives equal to or greater than 20 lines away from the `#if` or `#ifdef` directive to which they are related shall be decorated by language comment indicating the conditional they are associated with.